modules/local/get_test_data.nf

```diff
@@ -1,13 +1,23 @@
 process GET_TEST_DATA {
+    label "process_single"
+
     container "docker.io/yuukiiwa/git:latest"
 
     output:
-    path "test-datasets/fast5/$barcoded/*"  , emit: ch_input_fast5s_path
-    path "test-datasets/modification_fast5_fastq/"   , emit: ch_input_dir_path
+    path "test-datasets/fast5/$barcoded/*"        , emit: ch_input_fast5s_path
+    path "test-datasets/modification_fast5_fastq/", emit: ch_input_dir_path
+    path "versions.yml"                           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
 
     script:
     barcoded = (workflow.profile.contains('test_bc_nodx') || workflow.profile.contains('rnamod')) ? "nonbarcoded" : "barcoded"
     """
     git clone https://github.com/nf-core/test-datasets.git --branch nanoseq --single-branch
```

Comment:
I'm assuming this is for CI testing, right?
We can do so directly in the GHA, or locally if needed.
cf methylseq: https://github.com/nf-core/methylseq/blob/93bc5811603c287c766a0ff7e03b5b41f4483895/.github/workflows/ci.yml#L45-L61

Reply:
Will address this in the 3.2 release